### PR TITLE
feat: Add Boba Register package & Sepolia addresses

### PR DESCRIPTION
**Pull request by wsdt**
# Changes
- Moved bobanetwork/register package over to v3-anchorage
- Added Sepolia addresses based on: https://github.com/bobanetwork/v3-anchorage/tree/develop/packages/contracts-bedrock/deployments/sepolia
- Released new @bobanetwork/register version (including Sepolia addresses and removing Avax addresses) to integrate it into the gateway.

### CODE REVIEW

File: packages/boba/register/addresses/layerZeroTestnet.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+  "Layer_Zero_Protocol": {
```
Comment (wsdt) [inline packages/boba/register/addresses/layerZeroTestnet.json:2]: I guess these change also - do we have those deployed on Sepolia already? @Boyuan-Chen

**Comment by wsdt** [comment]
Used addresses from: https://github.com/bobanetwork/v3-anchorage/pull/99 and https://github.com/bobanetwork/v3-anchorage/pull/102

File: packages/boba/register/addresses/layerZeroTestnet.json (hunk 1 above)
Comment (boyuan-chen) [reply packages/boba/register/addresses/layerZeroTestnet.json:2]: No

**Comment by boyuan-chen** [comment]
Please update L2OutputOracle address to the new one.

File: packages/boba/register/.env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Local
+L1_NODE_WEB3_URL=http://localhost:9545
```
Comment (souradeep-das) [inline packages/boba/register/.env.example]:
```suggestion
L1_NODE_WEB3_URL=http://localhost:8545
```

File: packages/boba/register/README.md
Comment (souradeep-das) [inline packages/boba/register/README.md:1]: this readme looks like its outdated (deploy processes)

File: packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json
Comment (souradeep-das) [inline packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json:1]: do we need this file?

File: packages/boba/register/hardhat.config.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+import '@nomiclabs/hardhat-waffle'
+import './tasks/deploy'
+
+const config: HardhatUserConfig = {
+  mocha: {
+    timeout: 300000,
+  },
+  networks: {
+    boba: {
+      url: 'http://localhost:8545',
```
Comment (souradeep-das) [inline packages/boba/register/hardhat.config.ts]:
```suggestion
      url: 'http://localhost:9545',
```

File: packages/boba/register/hardhat.config.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      saveDeployments: false,
+    },
+    localhost: {
+      url: 'http://localhost:9545',
```
Comment (souradeep-das) [inline packages/boba/register/hardhat.config.ts]:
```suggestion
      url: 'http://localhost:8545',
```

**Review by souradeep-das — APPROVED** [review]
lgtm!

File: packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json
Comment (wsdt) [reply packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json:1]: Not necessarily, but it's also included in the other PRs. Can remove it if we are sure we won't use them.

File: packages/boba/register/README.md
Comment (wsdt) [reply packages/boba/register/README.md:1]: actually just copied it over from the old boba repo. Should be the same deploy.ts task. 

Looks like we used to have contracts in there. Not sure how we deploy these now @Boyuan-Chen ? 
Can I remove all the hardhat files etc? On my side we just need to package the address json's as npm package.

File: packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json
Comment (souradeep-das) [reply packages/boba/register/addresses/addressesSepolia_0x9bFE9c5609311DF1c011c47642253B78a4f33F4B.json:1]: These addresses have their source from upstream (optimism's older depl). I think we should be good removing it from our register package :)

File: packages/boba/register/README.md
Comment (boyuan-chen) [reply packages/boba/register/README.md:1]: Yes, please remove it.

**Comment by wsdt** [comment]
@souradeep-das @Boyuan-Chen done! thx!

**Review by sk-enya — APPROVED** [review]
LGTM!

**Pull request merged**